docs/Analyzer Configuration.md

```diff
@@ -108,6 +108,29 @@ Examples:
 |`dotnet_code_quality.null_check_validation_methods = NS.MyType.Validate(ParamType)` | Matches specific method 'Validate' with given fully qualified signature
 |`dotnet_code_quality.null_check_validation_methods = NS1.MyType1.Validate1(ParamType)~NS2.MyType2.Validate2(ParamType)` | Matches specific methods 'Validate1' and 'Validate2' with respective fully qualified signature
  
+### Excluded symbol names
```

Comment:
NOTE: I have purposefully kept the name very generic here as it is likely we want to support this option for more rules  in future where users want to skip analysis for certain symbols.